editor/src/components/canvas/dom-walker.spec.browser.tsx

```diff
@@ -143,10 +143,9 @@ describe('DOM Walker tests', () => {
           <Storyboard data-uid={'${BakedInStoryboardUID}'}>
             <Scene
               static
-              style={{ left: 0, top: 0, width: 375, height: 812 }}
+              style={{ position: 'relative', left: 0, top: 0, width: 375, height: 812 }}
```

Comment:
Shouldn't these all be `position: 'absolute'`?

Reply:
I thought so too, but it turns out utopia-api added relative position to pinsystem parents, and absolute position to all children. I tried to match the current tests with how the pragma works.

Reply:
hm I also think it makes more sense as position: 'absolute' because otherwise sibling scenes will muck things up

Reply:
fixed :) 

Maybe in the past even storyboard had the pinSystem set so all Scenes were absolute?